osgar/lib/config.py

```diff
@@ -14,7 +18,20 @@ class Config(object):
 
     @classmethod
     def load(cls, filename):
-        return cls.loads(open(filename).read())
+        if isinstance(filename, list):
+            filenames = filename
+        else:
+            filenames = [filename]
+        
+        ret = Config()
+        for filename in filenames:
+            with open(filename) as f:
+                c = cls.loads(f.read())
+                if ret is None:
+                    ret.data = c.copy()
+                else:
+                    ret.data = Config.merge_dict(ret.data, c.data)
```

Comment:
Oh my. Here you are shadowing the class attribute `data` with instance attribute.

Reply:
This is bug. I was going to write that I am directly using new instance `ret = Config()`, but there should be `ret.data = c.data.copy()`.

Reply:
well, the `ret` is actually never `None` ... it is bit of code when `update()` was used.